lib/json/parser.js

```diff
@@ -23,12 +23,17 @@ function translateStructure(structure, shape) {
 
   var struct = {};
   var shapeMembers = shape.members;
+  var isAwsQueryCompatible = shape.api && shape.api.awsQueryCompatible;
   util.each(shapeMembers, function(name, memberShape) {
     var locationName = memberShape.isLocationName ? memberShape.name : name;
     if (Object.prototype.hasOwnProperty.call(structure, locationName)) {
       var value = structure[locationName];
       var result = translate(value, memberShape);
       if (result !== undefined) struct[name] = result;
+    } else if (isAwsQueryCompatible && memberShape.defaultValue) {
+      if (memberShape.type === 'list') {
+        struct[name] = typeof memberShape.defaultValue === 'function' ? memberShape.defaultValue() : memberShape.defaultValue;
+      }
```

Comment:
the reason we only apply this to lists and not maps is because the XML behavior was only applied to lists